### PR TITLE
Save transformed components for WA templates

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/utils/whatsapp/tests.py [hunk 1]
```diff
@@ -233,11 +239,11 @@ def test_update_local_templates_whatsapp(self):
         self.assertEqual(TemplateTranslation.STATUS_UNSUPPORTED_COMPONENTS, ct.status)
         self.assertEqual("foo_namespace", ct.namespace)
         self.assertEqual(
-            [
-                {"add_security_recommendation": "True", "type": "BODY"},
-                {"type": "FOOTER"},
-                {"buttons": [{"otp_type": "COPY_CODE", "text": "copy", "type": "OTP"}], "type": "BUTTONS"},
-            ],
+            {
+                "body": {"content": "", "params": []},
```
Comment (rowanseymour) [inline temba/utils/whatsapp/tests.py:243]: little confused as to why we have components here with no content or params..

Comment (norkans7) [reply temba/utils/whatsapp/tests.py:243]: There are template for OTP, they have components such as 

https://github.com/nyaruka/rapidpro/pull/5042/files#diff-766c88132c33459d25a1214a036b91efbe9f5ce2239ba46c243cdb9d795e552cR169-R172

Comment (rowanseymour) [reply temba/utils/whatsapp/tests.py:243]: but if there's no params or content... should we include them here?

Comment (norkans7) [reply temba/utils/whatsapp/tests.py:243]: We'll probably not support those to be used but we still sync them to avoid confusion

**Review by rowanseymour — APPROVED** [review]

**Pull request merged**